erpnext/selling/report/sales_person_commission_summary/sales_person_commission_summary.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2013, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe import msgprint, _
+from frappe.utils import flt
+
+
+def execute(filters=None):
+	if not filters: filters = {}
+
+	columns = get_columns(filters)
+	entries = get_entries(filters)
+	data = []
+
+	for d in entries:
+			data.append([
+				d.name, d.customer, d.territory, d.posting_date,
+				d.base_net_amount, d.sales_person, d.allocated_percentage, d.allocated_amount,d.incentives
+			])
+
+	if data:
+		total_row = [""]*len(data[0])
+		data.append(total_row)
+
+	return columns, data
+
+def get_columns(filters):
+	if not filters.get("doc_type"):
+		msgprint(_("Please select the document type first"), raise_exception=1)
+
+	return [filters["doc_type"] + ":Link/" + filters["doc_type"] + ":140",
+		_("Customer") + ":Link/Customer:140", _("Territory") + ":Link/Territory:100", _("Posting Date") + ":Date:100",
+		_("Amount") + ":Currency:120",
+		_("Sales Person") + ":Link/Sales Person:140", _("Contribution %") + "::110",
+		_("Contribution Amount") + ":Currency:140",_("Incentives") + ":Currency:140"]
```

Comment:
Add "Commission Rate" column

Reply:
use the new `dict` style column definitions !